### ISSUE TITLE
[NuGet.org Bug]: [a11y compliance] Reserved checkmarks need a role/title/alt/etc attribute

### ISSUE THREAD

**Issue by drewgillies** [NuGet/NuGetGallery#8743]
### Impact

Other

### Describe the bug

To comply with screen readers etc, the guidance is:

Fix ONE of the following:
- Element does not have an alt attribute
- aria-label attribute does not exist or is empty
- aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
- Element has an empty title attribute
- Element's default semantics were not overridden with role="none" or role="presentation"

The reserved checkmarks in the Package Details page, and presumably other locations do not comply. 

### Repro Steps

Visit a Package Details page and view source--the required attributes are not present. Windows Narrator does work though.

### Expected Behavior

See above.

**Comment by drewgillies** [comment]
Note that this was tested on Windows Narrator and NVDA: https://github.com/NuGet/NuGetGallery/pull/8521 but still falls short of compliance with the required tags.
